src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -10,3 +10,10 @@
     [vector-icons/icon :icons/exclamation_mark {:color           :white
                                                 :container-style styles/error-exclamation}]
     [react/text {:style error-message-style} (i18n/label :t/wallet-error)]]])
+
+(defn wallet-syncing [error-container-style error-message-style]
```

Comment:
Code duplication with method above. This could be refactored to extract only differences.